webcomponents/tool/src/main/frontend/packages/sakai-grader/src/sakai-grader-rendering-mixin.js

```diff
@@ -232,6 +227,18 @@ export const graderRenderingMixin = Base => class extends Base {
                 </span>`;
   }
 
+  _renderGraderLinkBlock() {
+
+    return html`<div id="grader-link-block" class="float-end">
+            <button class="btn btn-primary active"
+              data-bs-toggle="offcanvas"
```

Comment:
the reason I push these attributes in four spaces is so that the actual content of the tag is more apparent. No biggy though.

Reply:
makes sense